lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -691,7 +691,8 @@ def commit(request, action, amount, options)
         end
 
         success = response[:decision] == "ACCEPT"
-        message = @@response_codes[('r' + response[:reasonCode]).to_sym] rescue response[:message]
+        message = response[:message]
```

Comment:
the normalized error message logic was moved here: https://github.com/activemerchant/active_merchant/pull/2423/files#diff-9f21097553e010a708d707ede0637ca8R744